Code/Framework/AzCore/AzCore/Memory/SystemAllocator.cpp

```diff
@@ -21,7 +21,7 @@
 
 #if !defined(AZCORE_SYSTEM_ALLOCATOR)
 // define the default
-#define AZCORE_SYSTEM_ALLOCATOR AZCORE_SYSTEM_ALLOCATOR_HPHA
+#define AZCORE_SYSTEM_ALLOCATOR AZCORE_SYSTEM_ALLOCATOR_MALLOC
```

Comment:
Do we have a description on the pros and cons of switching to the Malloc by default.
Perhaps we can update the PR description to indicate why were are making this change?

Reply:
I'm doing this just to understand the memory characteristics used by AR when using default MALLOC over HPHA first.

Reply:
That sounds fine to me.
Your comment now provides context from anyone reading this PR on why the change is occurring.
So we have an audit trail to follow, in case someone has to trace back through old commits.